geoportal/geomapfish_geoportal/static-ngeo/js/apps/Controllerdesktop_alt.js

```diff
@@ -57,23 +55,6 @@ class Controller extends AbstractDesktopController {
     if (this.dimensions.FLOOR == undefined) {
       this.dimensions.FLOOR = '*';
     }
-
-    /**
-     * @type {boolean}
-     */
-    this.drawLidarprofilePanelActive = false;
-
-    const drawLidarprofilePanelActive = new ngeoMiscToolActivate(this, 'drawLidarprofilePanelActive');
-    this.ngeoToolActivateMgr.registerTool('mapTools', drawLidarprofilePanelActive, false);
-    const $timeout = $injector.get('$timeout');
-
-    panels.getActiveToolPanel().subscribe({
-      next: (panel) => {
-        this.drawLidarprofilePanelActive = panel === 'lidar';
-
-        $timeout(() => {}); // this triggered on DOM click, we call $timeout to force Angular digest
-      },
-    });
```

Comment:
It should be removed [here](https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/apps/desktop_alt/Controller.js#L61), otherwise the updates will roll back this change